### DISCUSSION TITLE
No improvement with true recombination rate

### DISCUSSION THREAD

**Discussion by XingerTang** [AlphaGenes/AlphaPeel#242] [General]
@gregorgorjanc @AprilYUZhang
I tried to use the true recombination rate to generate the transmission function of the second Baum-Welch implementation in the peeling cycle, and it turned out no improvement in the accuracy compared to the previous equal recombination rate across all loci transmission rate.
This is weird, I suspect there is something off with the second Baum-Welch implementation, yet by removing the second Baum-Welch implementation, then all of the accuracies dropped significantly.

**Comment by XingerTang** [comment]
@AprilYUZhang Please find recombination rate from Martin Johnson's paper and let's resimulate the data with that recombination rate and compare using naive uniform rec rate in AlphaPeel with the realised recombination rate. 
If the accuracy is the same under that setting, then there is no point to use improved recombination rate in the peeling program. But we are still interested in using rec input from user.

**Comment by gregorgorjanc** [comment]
Have we really simulated this well? I think we have, but it would be good to have this simulation code stored somewhere if we ever want to get back to this. Who did this simulation @XingerTang @AprilYUZhang and where is the code for it?
